evm-bridging/main.go

```diff
@@ -263,57 +319,52 @@ func generateEncodedInitializeFunctionCall(
 	cadenceNFTIdentifier,
 	contractMetadata string,
 ) string {
-	//Run 'cast abi-encode'
-	cmd := exec.Command(
-		"cast",
-		"abi-encode",
-		`initialize(address,address,address,string,string,string,string,string,string)`,
-		owner,
-		underlyingNftContractAddress,
-		vmBridgeAddress,
+	const contractABI = `[{
+	"name": "initialize",
+	"type": "function",
+	"inputs": [
+		{"name": "owner", "type": "address"},
+		{"name": "underlyingNftContractAddress", "type": "address"},
+		{"name": "vmBridgeAddress", "type": "address"},
+		{"name": "name", "type": "string"},
+		{"name": "symbol", "type": "string"},
+		{"name": "baseTokenURI", "type": "string"},
```

Comment:
just fyi as it doesn't seem to be making a difference, names differ here too from those in the contract: `baseTokenURI_`, `__cadenceNFTAddress`, `_cadenceNFTIdentifier`, `_contractMetadata`